### ISSUE TITLE
megahit_sdbg_build: Assertion failed [Exit code -6]

### ISSUE THREAD

**Issue by Quicken-up** [voutcn/megahit#108]
I am trying to run v1.1.1 and get the same error when using the point release binary and when compiling from source. I am running through qsub in an HPC environment. I couldn't test on the human gut dataset as I get 403 forbidden errors when trying to download the files with wget. I have tried on a couple of our files with different numbers of threads and amounts of memory. Same error every time. Any help appreciated. Example log file output from a run with 2x128GB below:

[megahit_log.txt](https://github.com/voutcn/megahit/files/693811/megahit_log.txt)

**Comment by voutcn** [comment]
Could you list the files in `s1.megahit_asm2/tmp/k21/`? If `21.edges.info` exists, please attach it, thanks.

**Comment by Quicken-up** [comment]
-rw-r--r-- 1 .. 2.4M Jan  9 15:04 21.cand
-rw-r--r-- 1 ..    0 Jan  9 15:04 21.counting
-rw-r--r-- 1 .. 648M Jan  9 14:54 21.edges.0
-rw-r--r-- 1 .. 648M Jan  9 14:54 21.edges.1
-rw-r--r-- 1 ..    0 Jan  9 15:04 21.edges.info

**Comment by voutcn** [comment]
Both `21.edges.info` and `21.counting` are empty, and that's strange.
BTW, what does it mean by 2x128GB?

**Comment by Quicken-up** [comment]
I am submitting the jobs to a cluster via a bash script using qsub. 2x128GB means that in the bash script to run the job I specified the number of cores as 2, and the memory per core as 128G. 

I found the problem, which was caused by disk quotas. Having changed the running directory to one with more disk quota, it seems to be running fine now. Hopefully the details above will help others with a similar problem.
